### ISSUE TITLE
Demo is not working

### ISSUE THREAD

**Issue by PatrykSzwer** [Juicy/juicy-highlight#11]
Seems like demo for this element is not working:

![repcontent](https://user-images.githubusercontent.com/25042256/35103804-4534ec1a-fc67-11e7-9fdc-f831040bf834.gif)

Happens in both Chrome and Firefox.

cc: @alshakero

**Comment by warpech** [comment]
Do you mean locally or online? If online, then this depends on: https://github.com/Juicy/juicy.github.io/issues/7

**Comment by PatrykSzwer** [comment]
Yes, online. Didn't know that there is another repository for that. Closing as duplicate.

**Comment by PatrykSzwer** [comment]
Yes, online. (closed by mistake)

**Comment by alshakero** [comment]
Should be working now.

http://juicy.github.io/juicy-highlight/

**Comment by warpech** [comment]
@alshakero what exactly did change? I can see that you pushed code to a branch that is no longer used: https://github.com/Juicy/juicy-highlight/commits/gh-pages

**Comment by PatrykSzwer** [comment]
TBH I'm not sure. Those labels on buttons suggests some different behaviors, but for all of them It's the same (highlight after click). Shouldn't be like that, when You hover on a middle button it should highlight and change lights without click etc.? (I have no idea what this dirty suggestion "Play with me" means) Or by highlight You mean only this border, not the light changes after click? 
![repro](https://user-images.githubusercontent.com/25042256/35144421-c1af6d40-fd04-11e7-93dd-259fc7697714.gif)

**Comment by alshakero** [comment]
@PatrykSzwer you might be right. But I checked out v1.0.0 demos and they did the same. @tomalec should have the final say.


> @alshakero what exactly did change? I can see that you pushed code to a branch that is no longer used: gh-pages (commits)

I assumed the website was in GH pages following the most common pattern. But turns out not. So I fixed `master` It was a one character issue:

https://github.com/Juicy/juicy-highlight/commit/6c1894f80f23e4a8ff3140198a7ab1560ad85777#diff-eacf331f0ffc35d4b482f1d15a887d3bR9

**Comment by tomalec** [comment]
Hmm.. I don't remember the kind of behavior we had for "Play with me" ;) I'd leave it as it is, but for sure we need to update https://github.com/Juicy/juicy.github.io/issues/7

**Comment by warpech** [comment]
The demo looks as intended, but that change should have been a PR. @alshakero can you please make a new release?

**Comment by alshakero** [comment]
> @alshakero can you please make a new release?

I think there is a minor misunderstanding. Due to a mistake of mine, I didn't bump `bower.json` before releasing 2.0.0. And as we agreed on Slack, I:
1. deleted 2.0.0 tag (since bower doesn't mind that).
2. bumped `bower.json` for fix my mistake.
3. While I was on it I found a minor bug (that survived the review of the last PR) see here https://github.com/Juicy/juicy-highlight/commit/6c1894f80f23e4a8ff3140198a7ab1560ad85777#diff-6aef979b1c6a48b804b0ae1d54d9873e.
4. I found Patryk's issue about the demo (it took exactly 1 character to fix it), so I fixed it.
5. I released a new tag 2.0.0, with the my mistake's fix, the bug fix, and the working demo.

i.e tag 2.0.0 now contains all the fixes and a working demo, should I still release?
